path.go

```diff
@@ -259,12 +259,12 @@ func addParameterMetaInfo(segs []*routeSegment) []*routeSegment {
 		}
 	}
 
-	// loop from begin to end
+	// loop from beginning to end
 	for i := range segLen {
 		// check how often the compare part is in the following const parts
 		if segs[i].IsParam {
-			// check if parameter segments are directly after each other and if one of them is greedy
-			// in case the next parameter or the current parameter is not a wildcard it's not greedy, we only want one character
+			// check if parameter segments are directly after each other;
+			// when neither this parameter nor the next parameter are greedy, we only want one character
```

Comment:
I mentioned this in https://discord.com/channels/704680098577514527/704680099320037448/1430287466845307004